### ISSUE TITLE
Resample .zarr Cube

### ISSUE THREAD

**Issue by joaofelipecsantos** [dcs4cop/xcube#212]
**Describe the bug**
Error during the xcube resample for Sentinel-2 dataset.

**To Reproduce**
1. Generate the Sentinel-2 MSI irregular time cube using the `xcube gen`.

To generate it I took the netCDF files (tiles 31UET, 31UES, 31UDT, 31UDS).

**I use the config parameters:**
![TICKET01_01](https://user-images.githubusercontent.com/32440152/67864720-58e14c80-fb26-11e9-91db-e0fcaa0c152f.png)

The **output_size** and **output_region** I took from `xcube grid abox`:
![TICKET01_02](https://user-images.githubusercontent.com/32440152/67864773-77474800-fb26-11e9-8526-7855d69ab2b5.png)

**The .zarr cube was created with no errors.**
![ticket01_06](https://user-images.githubusercontent.com/32440152/67864968-c7bea580-fb26-11e9-87da-cf69c515be50.png)

**2. Run the `xcube resample` with the command line:**
![ticket01_03](https://user-images.githubusercontent.com/32440152/67865040-e9b82800-fb26-11e9-990c-8d9881cb53bd.png)

**HERE IS THE ERROR MESSAGE:**
![xcube_ticket01_resample_issue_with_chunk](https://user-images.githubusercontent.com/32440152/67864136-71049c00-fb25-11e9-8e33-5c156fd7f554.png)

**Run again the `xcube resample` but now with output as netCDF4:**
![ticket01_04](https://user-images.githubusercontent.com/32440152/67865109-0f453180-fb27-11e9-95e4-bbf64fc17b46.png)

**With output as netCDF4 the resample successfully finished.**
![ticket01_05](https://user-images.githubusercontent.com/32440152/67865248-461b4780-fb27-11e9-8c0d-54539ca9e651.png)

**Expected behavior**
I was expected to resample using as output the .zarr format (netCDF4 is too big and too slow).

**Screenshots**
Added during the text.

**Desktop (please complete the following information):**
 - OS: Linux Ubuntu 18.04
 - Browser: Firefox (not necessary)
 - Version: concerning the xcube version I updated the xcube and xcube-gen-bc before start (30/10/2019)

**Additional context**
I used the same `xcube resample` before for the Sentinel-3 (full North Sea) and it worked perfect to generate the 1D resample cube in .zarr format. **I think my problem is due the chunk option.**
What should I do to edit the chunk size to a *ideal match*.

Thank you for the help!!

**Comment by forman** [comment]
@joaofelipecsantos, thanks for the report! 

My hope is. that we just need to update `xcube resample` to rechunk the cube along the time axis. We'll check...

**Comment by forman** [comment]
What does `xcube --version` say?

**Comment by joaofelipecsantos** [comment]
Hello Norman,

My `xcube --version` says:
**xcube, version 0.3.0dev0**

I did another tests and I have two ideas for the issue (just ideas):
  - When I replace a file instead append I have problems to resample.
  - When I have some file dropped due to errors during the ingestion I have problems to resample.

Because I did a `xcube resample` using .zarr as output for other group of netCDF files and it worked fine.

Regards.
